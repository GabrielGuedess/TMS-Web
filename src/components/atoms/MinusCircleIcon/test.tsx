import { render } from '@testing-library/react';

import { MinusCircleIcon } from '.';

describe('<MinusCircleIcon />', () => {
  it('should render the heading', () => {
    const { container } = render(<MinusCircleIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
