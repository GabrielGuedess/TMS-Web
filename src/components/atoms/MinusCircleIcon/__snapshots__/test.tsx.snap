// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MinusCircleIcon /> should render the heading 1`] = `
<svg
  fill="none"
  height="24"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M11.92 22C17.42 22 21.92 17.5 21.92 12C21.92 6.5 17.42 2 11.92 2C6.42001 2 1.92001 6.5 1.92001 12C1.92001 17.5 6.42001 22 11.92 22Z"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
  <path
    d="M7.92001 12H15.92"
    stroke="currentColor"
    stroke-linecap="round"
    stroke-linejoin="round"
    stroke-width="1.5"
  />
</svg>
`;
